flepimop/gempyor_pkg/src/gempyor/utils.py

```diff
@@ -544,6 +666,18 @@ def create_resume_file_names_map(
     parquet file types and environmental conditions. The function adjusts the file name mappings
     based on the operational block index and the location of the last job output.
 
+    Args:
+        resume_discard_seeding:  Determines whether seeding-related file types should be included.
+        flepi_block_index: Determines a specific operational mode or block of the process.
+        resume_run_index: Resume run index.
+        flepi_prefix: File prefix.
+        flepi_slot_index: Index of the slot.
+        flepi_run_index: flepiMoP run index.
+        last_job_output: Adjusts the keys in the mapping to be prefixed with this path.
+
+    Return:
+        Dict[str, str]: A dictionary where keys are input file paths and values are corresponding
```

Comment:
Don't need the type hint here, it's in the function definition. Also `typing.Dict` has been deprecated since 3.9, let's switch to using `dict`: https://docs.python.org/3/library/typing.html#typing.Dict